docs/cli-options.rst

```diff
@@ -198,7 +198,13 @@ Instaloader to login.
 
 .. option:: --load-cookies BROWSER-NAME, -b BROWSER-NAME
 
-   Use Instagram cookie in your browser to login (works with :option:`--login`). Supported browsers: Chrome, Firefox, Edge, Opera, Safari, Brave.
+   Use Instagram cookie in your browser to login.
+   Compatible with :option:`--cookiefile` if you want to load cookies from browser profiles.
+   Incompatible with :option:`--login` due to potential username mismatch between user input and browser login.
+   Supported browsers: Chrome, Firefox, Edge, Opera, Safari, Brave.
+
+   After loading the cookies run the :option:`--login` option as it is required to download high quality media
+   and to make full use of instaloader's features.
```

Comment:
```suggestion
   and to make full use of Instaloader's features.
```

I think here we should also mention that this feature requires `browser_cookie3`.

Oh and could you also add the `.. versionadded:: 4.11` directive here?

Reply:
Done.